fitbenchmarking/resproc/visual_pages.py

```diff
@@ -175,17 +187,17 @@ def setup_page_misc(group_name, problem_name, res_obj, results_dir, count):
     @param count :: number of times a problem with the same name was
                     passed through this function, consecutively
 
-    @returns :: the directory in which the visual display pages go
-                the file path to the visual display page that is
-                currently being made
+    @returns :: the file path to the visual display page that is
```

Comment:
Do you think its worth changing the docstrings for this file now since we changed some things or wait till we look at #309?

Reply:
I'd rather leave it until #309 if you think the refactor will completely change it.